src/managers/KeybindManager.hpp

```diff
@@ -29,6 +29,7 @@ class CKeybindManager {
     std::list<SKeybind> m_lKeybinds;
 
     bool                handleInternalKeybinds(xkb_keysym_t);
+    static void         moveWindowToWorkspace(CWindow*, int);
```

Comment:
ayo sir what in the name of all thats holy is this

Reply:
That's the new method I mentioned with most of the code from `moveActiveToWorkspace`, since most of it is also needed by the `movewindowinv` dispatcher. Should I have put it somewhere else?

Reply:
no global methods there, move it to CCompositor

Reply:
Ah, I didn't realise it already existed there. Should I replace the existing CCompositor method (which doesn't seem to be used anywhere) or create a new one?

Reply:
do whatever you see fit